pooch/tests/test_downloaders.py

```diff
@@ -471,6 +475,7 @@ class TestZenodoAPISupport:
             ("invalid", invalid_api_response),
         ],
     )
+    @pytest.mark.network
```

Comment:
This test makes use of the `httpserver` pytest fixture provided by [`pytest-httpserver`](https://pypi.org/project/pytest_httpserver/), which runs a local http server for running this test. Therefore I don't think we need to add a `network` mark to it. Have you run in any problem trying to run this test without internet connection?
```suggestion
```

Reply:
Similar answer regarding `pytest-localftpserver`. Small difference: I didn't include the required `pytest` plugin. I'll do that and test again together with FTP tests.

Reply:
Cool. For reference, here's a list of all requirements for running the tests: https://github.com/fatiando/pooch/blob/main/env/requirements-test.txt

Reply:
Yeah. I just discovered that myself. I've taken over maintenance of `pooch` in Fedora recently. The previous maintainer used the PyPI sdist, which lacks the `env/` dir. I've switched to GitHub tarballs now. Makes life a little easier (apart from having to fight `setuptools_scm` :wink:).